src/microbe_stage/Microbe.Contact.cs

```diff
@@ -547,22 +552,26 @@ public void Kill()
 
             var sceneToUse = new ChunkConfiguration.ChunkScene();
 
-            // Try all organelles in random order and use the first one with a scene for model
-            foreach (var organelle in organelles.OrderBy(_ => random.Next()))
+            // Searches through the organellesAvailable list for a valid chunk to spawn
+            do
```

Comment:
Instead of using a `do` loop here, just checking if the enumerator can still move forward should be fine, as the enumerator not being valid is a permanent error that can't be fixed by looping more, so that should trigger the error condition print already.

Reply:
Or is this because organelles can have no model. In that case I think we still want to just run to the end of the enumerator once, and if we are out of organelle types to spawn, we should probably fallback to the cytoplasm model. If this fallback is used then changing HexCount to organelles.Count would also be unnecessary as the fallback would make sure that we can spawn as many chunks as is needed.